launchable/test_runners/maven.py

```diff
@@ -0,0 +1,24 @@
+import os
+import click
+from . import launchable
+
+
+@click.argument('source_roots', required=True, nargs=-1)
+@launchable.subset
+def subset(client, source_roots):
+    def file2test(f: str):
+        if f.endswith('.java'):
```

Comment:
Do you think we need to support `.kt` and `.scala` like our Gradle plugin, don't you? IIRC, Maven also can handle Scala and Kotlin.

Reply:
You're right. I hadn't considered that.
the maven is described in the [kotilin official documentation](https://kotlinlang.org/docs/reference/using-maven.html), so I'll fix the same with the gradle plugin

Reply:
fixed: https://github.com/launchableinc/cli/pull/118/commits/240905b6a645bb23bf26d4240be26f3a99513041